repositories/blog.repository.js

```diff
@@ -0,0 +1,107 @@
+const User = require("../models/user.model");
+const UserDTO = require("../DTO/user.dto");
+
+const getAllBlogs = async () => {
+  try {
+    const users = await User.findAll();
+
+    const dtoUsers = [];
+
+    users.forEach((user) => {
+      const dtoUser = new UserDTO(user);
+      dtoUsers.push(dtoUser);
+    });
+
+    return dtoUsers;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+};
```

Comment:
getAllBlogs returning users?

Reply:
I started to work in create blog issue because of my misunderstanding, for that reason I copy user repository, but this code is no longer in this issue. I deleted blog related files from this issue.